src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1723,7 +1728,7 @@ CHIP_ERROR CASESession::ValidateReceivedMessage(ExchangeContext * ec, const Payl
     else
     {
         mExchangeCtxt = ec;
-        mExchangeCtxt->SetResponseTimeout(kSigma_Response_Timeout + mExchangeCtxt->GetSessionHandle()->GetAckTimeout());
+        mExchangeCtxt->UseSuggestedResponseTimeout(kExpectedProcessingTime);
```

Comment:
And this sets the timeout for messages that are the response to Sigma1.

What sets the timeout for messages that are not the response to Sigma1 (e.g. for Sigma3)?  It looks to me like this PR will leave Sigma3 with a 5s timeout, which is wrong.

As in, I think this `UseSuggestedResponseTimeout` should be unconditional in `ValidateReceivedMessage`, and move to after this if/else that validates that `ec == mExchangeCtxt`.

Reply:
Why do we need two different models for processing times for Sigma messages? Why not use 5s everywhere?

Reply:
@mrjerryjohns Because lots of devices don't have hardware that can handle the crypto in Sigma1/Sigma2 in 5 seconds.

Reply:
I have updated the code as you suggest.